package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "uwazi",
-  "version": "1.170.0-rc1",
+  "version": "1.170.0-rc2",
```

Comment:
Please, take in mind that there is already a "1.170.0-rc2" version generated and deployed yesterday. So consider to upgrade this value to the proper one ("1.170.0-rc3").